.github/workflows/pkgci_build_packages.yml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Build Packages
+on:
+  workflow_call:
+    inputs:
+      package_version:
+        type: string
+        required: true
+
+jobs:
+  linux_x86_64_release_packages:
+    name: Linux Release Packages (x86_64)
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
+      - environment=prod
+      - cpu
+      - os-family=Linux
+    strategy:
+      fail-fast: false
+    env:
+      CACHE_DIR: ${{ github.workspace }}/.iree-container-cache
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          submodules: true
+      - name: Write version info
+        shell: bash
+        run: |
+          cat << EOF > version_info.json
+          {
+            "package-suffix": "",
+            "package-version": "${{ inputs.package_version }}",
+            "iree-revision": "$(cd ../iree && git rev-parse HEAD)",
+          }
+          EOF
+          realpath version_info.json
+          cat version_info.json
+      - name: Enable cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.CACHE_DIR }}
+          key: iree-pkgci-linux-release-x86_64-v1-${{ github.sha }}
+          restore-keys: |
+            iree-pkgci-linux-release-x86_64-v1-
+      - name: Build
+        run: |
+          export cache_dir="${{ env.CACHE_DIR }}"
+          export output_dir="${{ github.workspace }}/wheelhouse"
+          export toolchain_suffix=release
+          # If just iterating locally, uncomment this to build a cheap wheel.
+          # export packages="iree-runtime"
+          bash build_tools/pkgci/build_linux_packages.sh
```

Comment:
`shell: bash` should remove the need to prefix this command with `bash`

Reply:
There was no reason to not just execute it. Removed the prefix.